### PR TITLE
Base solution created

### PR DESCRIPTION
**Pull request by Vignesh-Mindtree**

### CODE REVIEW

File: Rpms.Ui/src/Rpms.Ui/bower.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "ASP.NET",
```
Comment (YashaswiniR) [inline Rpms.Ui/src/Rpms.Ui/bower.json]: Why name is set to ASP.NET?

File: Rpms.Ui/src/Rpms.Ui/package.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿{
+	"version": "1.0.0",
+	"name": "ASP.NET",
```
Comment (YashaswiniR) [inline Rpms.Ui/src/Rpms.Ui/package.json]: Why name is set to ASP.NET?

File: Rpms.Ui/src/Rpms.Ui/wwwroot/views/index.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+﻿<!DOCTYPE html>
+<html ng-app="">
+<head>
+    <meta charset="utf-8" />
+    <title></title>
+<script src="../lib/angular/angular.js"></script>
```
Comment (YashaswiniR) [inline Rpms.Ui/src/Rpms.Ui/wwwroot/views/index.html]: Do not load individual javascripts here. Use requirejs

File: Rpms.Ui/src/Rpms.Ui/wwwroot/views/index.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
+﻿<!DOCTYPE html>
+<html ng-app="">
```
Comment (YashaswiniR) [inline Rpms.Ui/src/Rpms.Ui/wwwroot/views/index.html]: Why is this empty?

File: Rpms.Ui/src/Rpms.Ui/bower.json (hunk 1 above)
Comment (Vignesh-Mindtree) [inline Rpms.Ui/src/Rpms.Ui/bower.json]: fixed

File: Rpms.Ui/src/Rpms.Ui/package.json (hunk 2 above)
Comment (Vignesh-Mindtree) [inline Rpms.Ui/src/Rpms.Ui/package.json]: fixed

File: Rpms.Ui/src/Rpms.Ui/wwwroot/views/index.html (hunk 3 above)
Comment (Vignesh-Mindtree) [inline Rpms.Ui/src/Rpms.Ui/wwwroot/views/index.html]: Fixed

File: Rpms.Ui/src/Rpms.Ui/wwwroot/views/index.html (hunk 4 above)
Comment (Vignesh-Mindtree) [inline Rpms.Ui/src/Rpms.Ui/wwwroot/views/index.html]: Fixed

**Pull request closed without merging**